dashboard/app/models/lti_integration.rb

```diff
@@ -22,11 +22,16 @@
 #  index_lti_integrations_on_platform_id  (platform_id)
 #
 class LtiIntegration < ApplicationRecord
-  validates :platform_id, presence: true
   validates :issuer, presence: true
   validates :client_id, presence: true
   validates :platform_name, presence: true
   validates :auth_redirect_url, presence: true
   validates :jwks_url, presence: true
   validates :access_token_url, presence: true
+
+  before_create :set_uuid
+
+  def set_uuid
+    self.platform_id = SecureRandom.uuid.delete "-"
```

Comment:
why are we stripping dashes out, rather than using a standard UUID?

Reply:
Really just to save space (minimal, I know) and just that they aren't necessary. These will be used as a unique url path for LMS clients in the (possibly rare) case there isn't a client id on an incoming request.

Reply:
Might be premature optimization... but having a standard for uuid's could be good to establish. I would think that `CHAR(36)` is most suitable for fixed-width uuid's, though `VARCHAR(36)` works too.

Reply:
If we really wanted to have an optimized solution that would work even for large tables where we do want to save some space and we want to stick to the UUID spec, we could do something like https://dev.mysql.com/blog-archive/storing-uuid-values-in-mysql-tables/ and store the UUIDs as binary. Otherwise, if we're ok with a simple opaque string it probably doesn't really matter if we do dashes or not.

Reply:
Sounds good. As an outcome of this conversation, I will:

1) Open a new PR that adds a new migration changing this column to `:string(36)` (varchar(36))
2) Update this PR with the changes from that migration